uber/templates/registration/index.html

```diff
@@ -106,7 +106,7 @@
 <form role="form" method="get" action="index">
 <input type="hidden" name="order" value="{{ order }}" />
 <div class="input-group">
-<input class="form-control" type="text" name="search_text" value="{{ search_text }}" />
+<input class="focus form-control" type="text" name="search_text" value="{{ search_text }}" />
```

Comment:
This class causes the form field to have the cursor in it by default, which is what we want for the attendee search on the main reg page.